macros/sql/pivot.sql

```diff
@@ -41,6 +42,7 @@ Arguments:
 {% macro pivot(column,
                values,
                alias=True,
+               slug=False,
```

Comment:
I would default this to True as well!

Reply:
I agree, but I didn't want to break anybody's existing implementations